theoplayer/api-reference/roku.md

```diff
@@ -276,6 +277,14 @@ The PlayerConfiguration object is passed to the configure method. It is an assoc
 | discoveryUrls     | Array of strings | Array of discovery URLs for your OptiView Live deployment. If `theoLive` config is omitted, the default URL is 'https://discovery.theo.live/v2/publications/'. |
 | externalSessionId | string           | A session ID to use for your OptiView Live session. This can tie an application session to a THEOlens session.                                                 |
 
+## OptiView Live API
+
+The OptiView Live API exposes the following property.
+
+| Property  | Type   | Description                                                                                                     |
+| --------- | ------ | --------------------------------------------------------------------------------------------------------------- |
+| authToken | string | Optional. The JWT to be included on OptiView Live media requests in the Authorization header as a bearer token. |
```

Comment:
nitpick: hey @turbidwater, OptiviewLive or TheoLive? The other PRs for JWTs still reference THEOlive, but I'm not sure tbh